packages/shared/src/WebView.js

```diff
@@ -5,13 +5,14 @@ import { WebView as NativeWebView } from 'react-native';
 import { Translation } from '@kiwicom/mobile-localization';
 
 import GeneralError from './errors/GeneralError';
+import type { WebViewStateChangeEvent } from '../types/Events';
 
-type Props = {
-  // not exact - may contain additional properties (extend as needed)
```

Comment:
This seemed to have been not exact on purpose; has this changed now?

Reply:
We had more components like this, but they are slowly disappearing. It is better to define what we can pass in, and expand when needed. In this case, we were only using source.uri as prop anyways.